src/legacy/core_plugins/metrics/public/_variables.scss

```diff
@@ -9,3 +9,5 @@ $tvbValueColorReversed: transparentize($euiColorEmptyShade, .2);
 
 $tvbHoverBackgroundColor: transparentize($euiColorFullShade, .9);
 $tvbHoverBackgroundColorReversed: transparentize($euiColorEmptyShade, .9);
+
+$tvbSplitBlockVisMinSize: 200px;
```

Comment:
Can you use a multiplier of an euiVariable here? `200` seems a little arbitrary, so maybe just `$euiSize * 10` = 160px